docker-multinode/common.sh

```diff
@@ -232,12 +227,14 @@ kube::multinode::restart_docker(){
         yum -y -q install bridge-utils
       fi
 
-      kube::helpers::file_replace_line ${DOCKER_CONF} `# Replace content in this file` \
-        "--bip" `# Find a line with this content...` \
-        "OPTIONS=\"\$OPTIONS --mtu=${FLANNEL_MTU} --bip=${FLANNEL_SUBNET}\"" `# ...and replace the found line with this line`
+      # Is there an uncommented OPTIONS line at all?
+      if [[ -z $(grep "OPTIONS" $DOCKER_CONF | grep -v "#") ]]; then
```

Comment:
How about

$(grep "^OPTIONS" $DOCKER_CONF)

Then it doesn't matter if a comment is on the end of the line. If you want to still do a grep -v, maybe

grep -v "^\s*#"

So that this line doesn't get missed

OPTIONS # some comment